### PR TITLE
Update URL in scripts

### PR DESCRIPTION
**Pull request by brianstrauch**
Release Notes
-------------

Breaking Changes
- PLACEHOLDER

New Features
- PLACEHOLDER

Bug Fixes
- PLACEHOLDER

Checklist
---------
- [x] Leave this box unchecked if features are not yet available in production

What
----

References
----------

Test & Review
-------------

### CODE REVIEW

**Review by sgagniere — APPROVED** [review]

**Review by sgagniere — CHANGES_REQUESTED** [review]
Looks like the installer is broken

File: install.sh [hunk 1]
```diff
@@ -412,17 +412,20 @@ main() {
...
-  NAME=${BINARY}_${VERSION}_${OS}_${ARCH}
-  TARBALL=${NAME}.${FORMAT}
-  TARBALL_URL=${S3_ARCHIVES_URL}/${TARBALL}
-  CHECKSUM=${BINARY}_${VERSION}_checksums.txt
-  CHECKSUM_URL=${S3_ARCHIVES_URL}/${CHECKSUM}
+
+  # As of v4.1.0, the base URL is packages.confluent.io and the archive/checksum names do not contain the version
+  ABOVE_4_0=$( { [ "${VERSION}" = "latest" ] || { [ "${VERSION#v2.}" = "${VERSION}" ] && [ "${VERSION#3.}" = "${VERSION}" ] && [ "${VERSION#4.0.}" = "${VERSION}" ]; }; } && echo "true" || echo "false")
+  BASEURL=$([ ${ABOVE_4_0} = "true" ] && echo "https://packages.confluent.io" || echo "https://s3-us-west-2.amazonaws.com/confluent.cloud")
+  NAME=$([ ${ABOVE_4_0} = "true" ] && echo "${BINARY}_${OS}_${ARCH}" || echo "${BINARY}_${VERSION}_${OS}_${ARCH}")
```
Comment (brianstrauch) [inline install.sh:423]:
```suggestion
  NAME=$([ ${ABOVE_4_0} = "true" ] && echo "${BINARY}_${OS}_${ARCH}" || echo "${BINARY}_${VERSION}_${OS}_${ARCH}")
  TARBALL=${NAME}.${FORMAT}
```

File: install.sh [hunk 2]
```diff
@@ -412,17 +412,20 @@ main() {
...
+  CHECKSUM=$([ ${ABOVE_4_0} = "true" ] && echo "${BINARY}_checksums.txt" || echo "${BINARY}_${VERSION}_checksums.txt")
+
+  ARCHIVES_URL=${BASEURL}/${PROJECT_NAME}/archives/${VERSION#v}
+  TARBALL_URL=${ARCHIVES_URL}/${NAME}.${FORMAT}
```
Comment (brianstrauch) [inline install.sh:427]:
```suggestion
  TARBALL_URL=${ARCHIVES_URL}/${TARBALL}
```

**Review by sgagniere — APPROVED** [review]
thanks!

**Pull request merged**